spec/integ/matrix-client-event-timeline.spec.js

```diff
@@ -538,6 +539,92 @@ describe("MatrixClient event timelines", function() {
             expect(timeline.getEvents().find(e => e.getId() === THREAD_ROOT.event_id)).toBeTruthy();
             expect(timeline.getEvents().find(e => e.getId() === THREAD_REPLY.event_id)).toBeTruthy();
         });
+
+        it("should return undefined when event is not within a thread but timelineSet is", async () => {
```

Comment:
The current title makes sense but I had to read it a couple times to get it.

Maybe this:

```suggestion
        it("should return undefined when event is not within the thread that the given timelineSet is representing", async () => {
```

Reply:
Addressed by https://github.com/matrix-org/matrix-js-sdk/pull/2454#discussion_r896284318